pyinterprod/interpro/contrib/gp.py

```diff
@@ -0,0 +1,40 @@
+import cx_Oracle
+
+
+def import_gp(cur: cx_Oracle.Cursor, file_path: str):
+    gp = []
+    with open(file_path, "rt") as fh:
+        for line in fh:
```

Comment:
Trim the new line character of each line:

```suggestion
        for line in map(str.rstrip, fh):
```